openedx/core/djangolib/net_utils.py

```diff
@@ -0,0 +1,24 @@
+"""
+Helper functions and classes for network utilities
+"""
+
+
+from ipware.ip2 import get_client_ip
+
+
+def get_ip_address(request, real_ip_only=False):
```

Comment:
Please split into two functions: `get_ip` and `get_real_ip`, with tests for both.